### DISCUSSION TITLE
New Student Developer Pack Partner: Notion!

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#137293] [GitHub Education]
![notion-logo](https://github.com/user-attachments/assets/81416dd2-fe8d-4ed4-94ad-bd8e6fc618b5)

We're thrilled to welcome Notion to the [Student Developer Pack](https://education.github.com/pack). Notion is a single space where you can think, write, and plan. Capture thoughts, manage projects, run your student org or hackathon and do much more, all in one place.

Notion and GitHub share the goal of empowering students with the tools they need to succeed in their studies and future careers. Together, we can support the next generation of thinkers, dreamers, and leaders. 
![notion documents](https://github.com/user-attachments/assets/f2560588-3ae9-4271-b632-16d407db3fea)

### Offer details
- Everything in the Notion Education plan plus additional AI responses. Notion Education plan offers everything in our Plus plan + increased sharing & collaboration capabilities and longer page history.
- You'll also find a dedicated template collection curated just for GitHub students that will grow and change with your needs.

Built right into your workspace, Notion AI is at the ready to help you brainstorm ideas, summarize, help you write, and find what you're looking for.

Did you know there was a GitHub x Notion integration? Notion's integration with GitHub allows you to seamlessly connect your development workflows: [explore the integration](https://www.notion.so/help/github).

You can also explore our [Marketplace](https://github.com/marketplace) for Notion x GitHub Actions [like this.](https://github.com/marketplace/actions/notion-x-github-action)

### Up Next
We're doing a stream with Notion on Wednesday 9/4 at 3pm EST on our [GitHub Education Twitch Channel](https://www.twitch.tv/githubeducation). Join us to learn more about the tool, how they support students going back to school and how you can utilize their AI feature to power your workspaces.

Have questions for the Notion team? Drop them in this thread and we'll try to get them answered on the stream!

[Get verified as a GitHub student](https://education.github.com/discount_requests/application) to make the most of your benefits in the Student Developer Pack today!

**Comment by Welding-Torch** [comment]
What about the duration of the Notion plus plan?

**Reply by morganersery1** [reply]
@Welding-Torch As long as your GitHub student coupon is active, you will have use of the Notion offer.

**Comment by md-ashraful-islam-ullas** [comment]
What's the duration of this plan?

**Reply by morganersery1** [reply]
@md-ashraful-islam-ullas As long as your GitHub student coupon is active, you will have use of the Notion offer.

**Comment by immanuel-peter** [comment]
@morganersery1 If I was paying separately for Notion AI previously, and we update our Notion plan to Education Plus through GitHub SDP, can I cancel my payments for Notion AI and still use Notion AI through the Education Plus plan from SDP?

Hopefully, that makes sense 😂.

**Reply by morganersery1** [reply]
@immanuel-peter  Yes that does make sense. If you cancel your AI add on you were paying for, you should still get use of Notion AI within the EDU plan. If it doesn't work, let Notion know (and tell us here) and we can get it sorted.

**Reply by immanuel-peter** [reply]
@morganersery1 Thanks! Do you also know which email address to contact Notion if anything does go awry?

**Reply by morganersery1** [reply]
Try: [team@makenotion.com](mailto:team@makenotion.com).

**Reply by immanuel-peter** [reply]
Okay, I emailed them about this 2-3 days ago, and they haven't replied.

**Reply by morganersery1** [reply]
I'm not sure what their response time is but that is the correct support email. Have you tried cancelling the add on and confirming that it still works? Once the system knows you've signed up as a part of the EDU plan through GitHub, you'll get the benefits attached to that offer, which include use of Notion AI.

**Reply by immanuel-peter** [reply]
It still works for now, because they only "remove" it after the month's billing cycle. Right now, it shows that I am on the Education Plus Plan from the SDP, so I hope Notion AI persists after next month.

**Reply by immanuel-peter** [reply]
@morganersery1 Notion AI was removed from my plan, but I am still on the Education Plus Plan from the SDP. Can anything be done to regain it?

**Reply by morganersery1** [reply]
@immanuel-peter you will first need to find out why they removed your plan. We don't have insight or access to their accounts and user information. If you are on the Edu Plus Plan from the SDP you do still get Notion AI credits. It is likely that you cannot have both attached to one account at the same time.

**Reply by immanuel-peter** [reply]
@morganersery1 I emailed the Notion Team, and they said that Notion AI was not included in the SDP. It's an add-on that has to be purchased separately. Is this true?

**Reply by morganersery1** [reply]
@immanuel-peter Notion AI credits are a part of the Edu offer via the SDP. Can you create a ticket with us (GitHub) and include any screenshots of what you're seeing both in the Notion platform and vis correspondence with their support team so that we can investigate further?

**Reply by immanuel-peter** [reply]
@morganersery1 Sure thing. Where can I submit the ticket?

**Reply by morganersery1** [reply]
https://support.github.com/contact

**Reply by immanuel-peter** [reply]
@morganersery1 I just submitted the ticket.

**Reply by morganersery1** [reply]
@immanuel-peter Thanks! I got it and assigned it to myself. The screenshots are super helpful to better understand your situation and help troubleshoot. I'll be sharing any updates on your situation in that support ticket directly.

**Comment by phf1001** [comment]
Hello! The link to the templates is not working. Can you update it please? :)

**Reply by morganersery1** [reply]
@phf1001  Are you seeing this link: https://www.notion.so/templates/collections/github-student-developer-pack ? It's the same one from the pack as well. Let me know if you're getting a 404 when clicking it.

**Reply by phf1001** [reply]
Exactly, that's the link I was seeing and indeed if gives me a 404

**Reply by morganersery1** [reply]
Hi @phf1001 Thanks for confirming. We're not able to recreate this error at this time, but I'll check in with Notion to see if anything is going on. Please clear your cache just in case. I'd also recommend going straight to the [main Collections](https://www.notion.so/templates/collections) page and clicking on the GitHub tile. Are other Notion pages giving you a 404?

**Reply by phf1001** [reply]
Oh, clearing the caché worked. Thanks!!

**Reply by digocoutinho78** [reply]
Thanks! :)
:smiley:

**Comment by Welding-Torch** [comment]
Hi, you've mentioned "Everything in the Notion Education plan plus additional AI responses". Can you specify how many additional AI responses? What's the number?

**Reply by morganersery1** [reply]
Hi @Welding-Torch When you redeem the offer your number of AI responses is reflected in your Notion account. Based on Notion's [fair usage](https://www.notion.so/help/ai-pricing-and-usage?_gl=1*iym0b*_gcl_au*NDI4ODkxODYuMTcxOTg3MjIzNQ..*_ga*MTU0NTMzODY1OC4xNzE5ODcyMjM3*_ga_9ZJ8CB186L*MTcyNzQ1MTkzMi4xNS4xLjE3Mjc0NTIxNDYuMjEuMC4w#ai-usage-limits) policy, this number can vary slightly, which is why there's not a definitive, X number of responses.

**Reply by Welding-Torch** [reply]
Hi, I found the answer by experimentation. Apparently it's 100 AI responses.
![image](https://github.com/user-attachments/assets/81ae4f1e-e770-4663-8925-4c0e788ef71a)

I do badly wish it was more than 100.

**Comment by asiu9** [comment]
how do i activate these benefits in my notion account ?

**Comment by kallmejoe** [comment]
notion no longer give GitHub integration for plus plans (aka students)
